lxc/storage.go

```diff
@@ -1017,9 +1039,84 @@ func (c *storageCmd) doStoragePoolVolumeShow(client lxd.ContainerServer, pool st
 	return nil
 }
 
+func (c *storageCmd) doStoragePoolVolumeCopy(client lxd.ContainerServer, src string, dst string, rm bool) error {
+	// validate both src and dst string
+	dstVolName, dstVolPool := c.parseVolume("", dst)
+	srcVolName, srcVolPool := c.parseVolume("", src)
+	if dstVolPool == "" || srcVolPool == "" {
+		defaultProfile, _, err := client.GetProfile("default")
+		if err != nil {
+			return fmt.Errorf("No storage pool for source or target volume specified")
+		}
+
+		rootDiskDevice := defaultProfile.Devices["root"]
+		if rootDiskDevice != nil &&
+			rootDiskDevice["pool"] != "" &&
+			rootDiskDevice["type"] == "disk" &&
+			rootDiskDevice["source"] == "" &&
+			rootDiskDevice["path"] == "/" {
+
+			if srcVolPool == "" {
+				srcVolPool = rootDiskDevice["pool"]
+			}
+
+			if dstVolPool == "" {
+				dstVolPool = rootDiskDevice["pool"]
+			}
+		} else {
+			return fmt.Errorf("No storage pool for source or target volume specified")
+		}
+	}
```

Comment:
What's all that logic above for? It looks like we're trying to magically determine a default storage pool or something, I don't really see a good reason to do that. When copying storage volumes, I'd expect the user always tell us the source and destination pools.